src/Resources/app/storefront/snippet/SnippetFile_en_GB.php

```diff
@@ -4,6 +4,9 @@
 
 use Shopware\Core\System\Snippet\Files\SnippetFileInterface;
 
+/**
+ * @SuppressWarnings(PHPMD.CamelCaseClassName)
```

Comment:
The same as https://github.com/notebooksbilliger/ShopWare6-MainTheme/pull/10/files#r416641904